### PR TITLE
Blender exporter: `opacity` should be `transparency`

### PR DESCRIPTION
**Pull request by yomotsu**
opacity value should be a  positive number.

https://github.com/mrdoob/three.js/issues/5923#issuecomment-70422720

### CODE REVIEW

**Comment by yomotsu** [comment]
er... `opacity` must be `transparency` correctly..

**Comment by mrdoob** [comment]
Even if this won't be backwards compatible, maybe it would be better to change the `Loader` so it looks for `opacity` instead of `transparency`.

**Comment by repsac** [comment]
When I was working on this exporter I normally went through the loaders and made a note on the keys each was looking for (the older exporter was off on a few as well). This may have been an oversight or even confusion on my part because I was also looking at the documentation and the docs list `opacity` http://threejs.org/docs/#Reference/Materials/Material as an attribute.

**Comment by yomotsu** [comment]
Thank you for your suggestion.
I changed the loader to accept both `opacity` for new JSON format and `transparency` for backwards compatible.

File: utils/exporters/blender/addons/io_three/exporter/api/material.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def normal_map(material):
 @_material
 def opacity(material):
     logger.debug('material.opacity(%s)', material)
-    return round(material.alpha - 1.0, 2);
+    return round( material.alpha, 2 );
```
Comment (mrdoob) [inline utils/exporters/blender/addons/io_three/exporter/api/material.py]: If we're now using `opacity`, shouldn't this code stay as it was?

Comment (repsac) [inline utils/exporters/blender/addons/io_three/exporter/api/material.py]: I think his fix is correct. I believe I made a mistake here and @WestLangley had it correct https://github.com/mrdoob/three.js/issues/5923#issuecomment-70424221

Comment (yomotsu) [inline utils/exporters/blender/addons/io_three/exporter/api/material.py]: to make sure, I tried both cases.

I made a material with transparency-Alpha 0.3 in blender.
screen shot 2015-01-21 at 6 30 36 am

then published with 3 type of exporters.

in exporter r69, transparency will be `"transparency": 0.30000001192092896`
```
				{
					"DbgColor": 15597568,
					"DbgIndex": 1,
					"DbgName": "red",
					"blending": "NormalBlending",
					"colorAmbient": [0.800000011920929, 0.0, 0.0],
					"colorDiffuse": [0.800000011920929, 0.0, 0.0],
					"colorEmissive": [0.0, 0.0, 0.0],
					"colorSpecular": [0.0, 0.0, 0.0],
					"depthTest": true,
					"depthWrite": true,
					"shading": "Lambert",
					"specularCoef": 50,
					"transparency": 0.30000001192092896,
					"transparent": true,
					"vertexColors": false
				}
```

in r70, opacity will be `"opacity": -0.7`
```
    },{
        "shading": "lambert",
        "colorAmbient": [0.800000011920929,0.0,0.0],
        "DbgIndex": 1,
        "depthWrite": true,
        "wireframe": false,
        "depthTest": true,
        "opacity": -0.7,
        "DbgName": "red",
        "visible": true,
        "colorEmissive": [0.0,0.0,0.0],
        "blending": "NormalBlending",
        "transparent": true,
        "doubleSided": true,
        "DbgColor": 15597568,
        "colorDiffuse": [0.800000011920929,0.0,0.0]
    }],
```

exporter of this pull request, opacity will be `"opacity": 0.3`
```
    },{
        "colorAmbient": [0.800000011920929,0.0,0.0],
        "transparent": true,
        "shading": "lambert",
        "DbgColor": 15597568,
        "colorDiffuse": [0.800000011920929,0.0,0.0],
        "blending": "NormalBlending",
        "depthTest": true,
        "visible": true,
        "depthWrite": true,
        "doubleSided": true,
        "colorEmissive": [0.0,0.0,0.0],
        "DbgName": "red",
        "DbgIndex": 1,
        "opacity": 0.3,
        "wireframe": false
    }],
```

**Comment by mrdoob** [comment]
I see! Looks good then.
Many thanks! ^^

**Pull request merged**